### PR TITLE
GRAM: parse yield expr

### PR DESCRIPTION
**Pull request by vlad20012**

### CODE REVIEW

File: src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt [hunk 1]
```diff
@@ -166,6 +166,7 @@ fun factory(name: String): RsStubElementType<*, *> = when (name) {
     "PATH_EXPR" -> RsExprStub.Type("PATH_EXPR", ::RsPathExprImpl)
     "RANGE_EXPR" -> RsExprStub.Type("RANGE_EXPR", ::RsRangeExprImpl)
     "RET_EXPR" -> RsExprStub.Type("RET_EXPR", ::RsRetExprImpl)
+    "YIELD_EXPR" -> RsExprStub.Type("RET_EXPR", ::RsRetExprImpl)
```
Comment (Undin) [inline src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: I think we should increase stub version because if this change

Comment (vlad20012) [reply src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: Fixed

File: src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt [hunk 2]
```diff
@@ -166,6 +166,7 @@ fun factory(name: String): RsStubElementType<*, *> = when (name) {
     "PATH_EXPR" -> RsExprStub.Type("PATH_EXPR", ::RsPathExprImpl)
     "RANGE_EXPR" -> RsExprStub.Type("RANGE_EXPR", ::RsRangeExprImpl)
     "RET_EXPR" -> RsExprStub.Type("RET_EXPR", ::RsRetExprImpl)
+    "YIELD_EXPR" -> RsExprStub.Type("YIELD_EXPR", ::RsRetExprImpl)
```
[the remark this answers is not in the document]
Comment (vlad20012) [reply src/main/kotlin/org/rust/lang/core/stubs/StubImplementations.kt]: 😞😞😞

**Comment by mchernyavsky** [comment]
Part of https://github.com/intellij-rust/intellij-rust/issues/2203.

**Pull request merged**